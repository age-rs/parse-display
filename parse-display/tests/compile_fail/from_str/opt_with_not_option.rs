use parse_display::FromStr;

#[derive(FromStr, PartialEq, Debug)]
struct X {
    #[display("a={}", opt)]
    a: u32,
}

fn main() {}
