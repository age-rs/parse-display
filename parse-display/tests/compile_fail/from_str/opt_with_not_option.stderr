error: field `a` is not a option type.
 --> tests/compile_fail/from_str/opt_with_not_option.rs:6:8
  |
6 |     a: u32,
  |        ^^^
